NiBAx/plugins/harmonization/harmonization.py

```diff
@@ -345,6 +347,43 @@ def OnAddToDataFrame(self):
         self.datamodel.data.loc[:,RAW_Residuals] = self.MUSE[RAW_Residuals]
         self.datamodel.data_changed.emit()
 
+
+    def OnMUSEharmonizationModelChanged(self):
+        if self.datamodel.IsValidHarmonization():
+            self.ui.Harmonization_Model_Loaded_Lbl.setHidden(False)
+            self.ui.Harmonization_Model_Loaded_Lbl.setObjectName('correct_label')
+            self.ui.Harmonization_Model_Loaded_Lbl.setStyleSheet('QLabel#correct_label {color: green}')
+            self.ui.Harmonization_Model_Loaded_Lbl.setText('Harmonization model compatible')
+            self.ui.Harmonized_Data_Information_Lbl.setObjectName('correct_label')
```

Comment:
If we need to specify an objectName, it is better for them to be unique. They don't have to be, but basic APIs from Qt to search objects by name will give errors. Plus it is less confusing when looking at the code.

Reply:
What do you suggest? Not to use object names or making them unique?